pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -77,6 +73,16 @@ func NewRemoteImageService(endpoint string, connectionTimeout time.Duration, tp
 			grpc.WithStreamInterceptor(otelgrpc.StreamClientInterceptor(tracingOpts...)))
 	}
 
+	connParams := grpc.ConnectParams{
+		Backoff: backoff.DefaultConfig,
+	}
```

Comment:
Any reason not to set MinConnectTimeout when initializing the struct?

Reply:
@tzneal i was not too sure about if someone else stomps on it (i think i tried it and it did not hold), so i ended up moving it just before DialContext